modules/search/client/views/search-sidebar-results.client.view.html

```diff
@@ -64,6 +64,7 @@ <h4>
         </span>
       </div>
       <div class="search-result-tagline"
+           ng-if="search.offer.type != 'meet'"
```

Comment:
You could use [one-time `::` binding](https://docs.angularjs.org/guide/expression#one-time-binding) here to make Angular faster (see other examples from this file)